v0.11.0/issue-template-gen/main.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Not sure why this file got included, but I assume we can remove (and maybe add to an exclude list somewhere?)

Reply:
This likely got included due to just copying the `docs/` directory in the `gen_versions.sh` script - should probably make that a little smarter in a different PR.